testing_utilities/approximate_quantity.hpp

```diff
@@ -131,43 +131,22 @@ template<typename Quantity>
 std::ostream& operator<<(std::ostream& out,
                          ApproximateQuantity<Quantity> const& q);
 
-// The 🄐 to 🄕 operators are only for hexadecimal literals.
-ApproximateQuantity<double> operator""_⑴(char const* representation);
-ApproximateQuantity<double> operator""_⑵(char const* representation);
-ApproximateQuantity<double> operator""_⑶(char const* representation);
-ApproximateQuantity<double> operator""_⑷(char const* representation);
-ApproximateQuantity<double> operator""_⑸(char const* representation);
-ApproximateQuantity<double> operator""_⑹(char const* representation);
-ApproximateQuantity<double> operator""_⑺(char const* representation);
-ApproximateQuantity<double> operator""_⑻(char const* representation);
-ApproximateQuantity<double> operator""_⑼(char const* representation);
-ApproximateQuantity<double> operator""_🄐(char const* representation);
-ApproximateQuantity<double> operator""_🄑(char const* representation);
-ApproximateQuantity<double> operator""_🄒(char const* representation);
-ApproximateQuantity<double> operator""_🄓(char const* representation);
-ApproximateQuantity<double> operator""_🄔(char const* representation);
-ApproximateQuantity<double> operator""_🄕(char const* representation);
+// The internal struct |NumericLiteral| and the operator""_ are to be used in
+// concert to allow the syntax 1.234_(1), or, in principle, 1.234_(123) for
+// approximate quantities.
+struct NumericLiteral {
+  ApproximateQuantity<double> operator(int);
+  char const* representation;
+};
+
+ApproximateQuantity<double> operator""_(char const* representation);
```

Comment:
Should this return `NumericLiteral`?  As written this function has no definition.